examples/azure/10-ark-config.yaml

```diff
@@ -24,9 +24,7 @@ persistentVolumeProvider:
     apiTimeout: <YOUR_TIMEOUT>
 backupStorageProvider:
   bucket: <YOUR_BUCKET>
-  azure:
-    location: <YOUR_LOCATION>
-    apiTimeout: <YOUR_TIMEOUT>
```

Comment:
do we not use the Azure API for object storage? or is it faster/doesn't require `apiTimeout` toggling?